msteams-platform/sbs-teams-conversation-bot.yml

```diff
@@ -50,6 +50,9 @@ items:
 
    [!INCLUDE [Set up local environment](includes/get-started/clone-repository.md)]
 
+   > [!div class="nextstepaction"]
+   > [I ran into an issue](https://github.com/MicrosoftDocs/msteams-docs/issues/new?template=Doc-Feedback.yaml&title=%5BI+ran+into+an+issue%5D+Set+up+local+environment&pageUrl=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fsbs-teams-conversation-bot%3Ftabs%3Ddev%26tutorial-step%3D2&contentSourceUrl=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fsbs-teams-conversation-bot.yml%23L64&documentVersionIndependentId=1c9cc6be-8ff3-de89-fa46-6bc355249626&author=surbhigupta&metadata=%2A%2BID%253A%2Be473e1f3-69f5-bcfa-bcab-54b098b59c80%2B%250A%2A%2BService%253A%2B%2A%2Amsteams%2A%2A)
```

Comment:
contentUrl is wrong:
https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/sbs-teams-conversation-bot.yml#L47

Reply:
Line number removed